basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,226 @@
 const helpers = require('./project-1');
 
 // start testing!
+
+/* 
+  multiplyByTen,
+  subtractFive,
+  areSameLength,
+  areEqual,
+  lessThanNinety,
+*/
+describe('1-5', () => {
+  it('multiplyByTen()', () => {
+    const expected = 1000;
+    const actual = helpers.multiplyByTen(100);    
+    expect(actual).toEqual(expected);
```

Comment:
Good, but this is your only case. What if your multiplyByTen() function looked like this?
```
function multiplyByTen(x) {
  return 1000;
}
```
Hey, look, I passed the only test 😄  Guess this function works as intended! In other words, make sure you're doing multiple tests.